### DISCUSSION TITLE
Transaction not received

### DISCUSSION THREAD

**Discussion by nobefrancois** [blockscout/blockscout#11014] [General]
Why has my transaction arrived in my wallet address because I entered my correct wallet address

**Comment by nobefrancois** [comment]
I have lost my funds in the transaction 😥

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
